data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -188,75 +188,55 @@ def prepare_germline_jobs(this_batch: Batch, this_batch_run: BatchRun, this_sqr:
-        is_manual = False
-        for sample_library_name in sample_library_names:
-            library_metadata: pd.DataFrame = metadata_df.query(f"sample=='{sample_library_name}'")
-            if not library_metadata.empty and library_metadata["workflow"].unique().item() == "manual":
-                logger.info(f"Skipping sample '{sample_name}'. Workflow column for matching "
-                            f"sample '{sample_library_name}' is set to manual")
-                is_manual = True
-                break
-
-        # Break out of manual
-        if is_manual:
+        if library_metadata["workflow"].unique().item() == "manual":
+            logger.info(f"Skipping sample '{sample_library_name}'. Workflow column for matching "
+                        f"sample '{sample_library_name}' is set to manual")
             # We do not pursue manual samples
-            logger.info(f"Skipping sample '{sample_name}'. "
+            logger.info(f"Skipping sample '{sample_library_name}'. "
                         f"Workflow column is set to manual for at least one library name")
             continue
 
         # iterate through libraries for this sample and collect their assay types
-        assay_types = []
-        for sample_library_name in sample_library_names:
-            library_metadata: pd.DataFrame = metadata_df.query(f"sample=='{sample_library_name}'")
-            if not library_metadata.empty:
-                assay_types.append(library_metadata["type"].unique().item())
-
-        # ensure there are some assay types for this sample
-        if len(set(assay_types)) == 0:
-            logger.warning(f"SKIP SAMPLE '{sample_name}' GERMLINE WORKFLOW LAUNCH. NO ASSAY TYPE METADATA FOUND.")
-            continue
-
-        # ensure only one assay type
-        if not len(set(assay_types)) == 1:
-            logger.warning(f"SKIP SAMPLE '{sample_name}' GERMLINE WORKFLOW LAUNCH. MULTIPLE ASSAY TYPES: {assay_types}")
-            continue
-
-        # now we assign this _single_ assay type
-        assay_type = list(set(assay_types))[0]
+        assay_type = library_metadata["type"].unique().item()
 
         # skip germline if assay type is not WGS
         if assay_type != "WGS":
-            logger.warning(f"SKIP {assay_type} SAMPLE '{sample_name}' GERMLINE WORKFLOW LAUNCH.")
+            logger.warning(f"SKIP {assay_type} SAMPLE '{sample_library_name}' GERMLINE WORKFLOW LAUNCH.")
             continue
 
         # convert read_1 and read_2 to cwl file location dict format
         sample_df["read_1"] = sample_df["read_1"].apply(cwl_file_path_as_string_to_dict)
         sample_df["read_2"] = sample_df["read_2"].apply(cwl_file_path_as_string_to_dict)
 
         job = {
-            "sample_name": sample_name,
-            "fastq_list_rows": sample_df.to_dict(orient="records"),
+            "sample_name": sample_library_name,
+            "fastq_list_rows": sample_df.drop(columns=["sample_library_name"]).to_dict(orient="records"),
```

Comment:
Dropping column name should be `sample_library_names`

```
"fastq_list_rows": sample_df.drop(columns=["sample_library_names"]).to_dict(orient="records"),
```

Reply:
Resolved with c88688c8b5212751a2e3703ffd6ada79ef6271a3